static/gen-static-ver

```diff
@@ -1,35 +1,39 @@
 #!/usr/bin/env bash
 
-ENGINE_DIR=$1
-VERSION=$2
+REPO_DIR="$1"
+VERSION="$2"
 
-if [ -z "$ENGINE_DIR" ] || [ -z "$VERSION" ]; then
-    echo 'usage: ./gen-static-ver ${ENGINE_DIR} ${VERSION}'
-    exit 1
+if [ -z "$REPO_DIR" ] || [ -z "$VERSION" ]; then
+	# shellcheck disable=SC2016
+	echo 'usage: ./gen-static-ver ${REPO_DIR} ${VERSION}'
+	exit 1
 fi
 
-DATE_COMMAND="date"
-if [[ $(uname) == "Darwin" ]]; then
-    DATE_COMMAND="docker run --rm alpine date"
-fi
-GIT_COMMAND="git -C $ENGINE_DIR"
+GIT_COMMAND="git -C $REPO_DIR"
 
 staticVersion="$VERSION"
 if [[ "$VERSION" == *-dev ]]; then
-    # based on golang's pseudo-version: https://groups.google.com/forum/#!topic/golang-dev/a5PqQuBljF4
```

Comment:
I'll look at extracting this code to a `gen-pseudo-version` (e.g.) script, as a follow-up. This PR was already big, so I decided to not make it even bigger for now